pmoves/services/agent-zero/mcp_server.py

```
@@ -273,7 +546,20 @@ def execute_command(cmd: Optional[str], payload: Optional[Dict[str, Any]] = None
+        return e2b_sandbox_create(payload)
+    if cmd == "e2b.sandbox.execute":
+        return e2b_sandbox_execute(payload)
+    if cmd == "e2b.sandbox.terminate":
+        return e2b_sandbox_terminate(payload)
+    if cmd == "e2b.desktop.create":
+        return e2b_desktop_create(payload)
+    if cmd == "e2b.spell.execute":
+        return e2b_spell_execute(payload)
+    if cmd == "e2b.surf.scrape":
+        return e2b_surf_scrape(payload)
+    raise ValueError(f"Unknown E2B command: {cmd}")
```

Ruff (0.14.10)

Avoid specifying long messages outside the exception class

(TRY003)